models/dfk/dfk_hero_summons.sql

```diff
@@ -0,0 +1,53 @@
+{{ 
+    config(
+        materialized='incremental',
+        unique_key='tx_hash',
+        incremental_strategy = 'delete+insert',
+        tags=['dfk'],
+        cluster_by=['block_timestamp']
+        ) 
+}}
+
+with
+
+logs as (
+    select
+        *
+    from {{ ref("logs") }}
+    where {{ incremental_load_filter("block_timestamp") }}
+),
+
+txs as (
+    select
+        *
+    from {{ ref("txs") }}
+    where {{ incremental_load_filter("block_timestamp") }}
+),
+
+crystal_to_hero_summons as (
+    select  
+        txs.block_timestamp,
+        txs.tx_hash,
+        substr(txs.data,11) as crystal_id,
+        txs.from_address as summoneer,
+        logs.event_inputs:tokenId as hero_id
+    from txs
+    left join logs on txs.tx_hash = logs.tx_hash and logs.event_name = 'Transfer'
```

Comment:
would recommend using join here instead of left join.
also separate join conditions on two lines